apps/client/src/features/viewers/minimal-timer/MinimalTimer.tsx

```diff
@@ -133,23 +134,13 @@ export default function MinimalTimer(props: MinimalTimerProps) {
   const isNegative = (time.current ?? 0) < 0;
   const showFinished = isNegative && !userOptions?.hideOvertime;
   const showEndMessage = time.current < 0 && general.endMessage && !hideEndMessage;
```

Comment:
remember that we will still need to check this. end message should never appear on a count up (I guess)